extensions/blocks/slideshow/index.js

```diff
@@ -116,6 +116,7 @@ export const settings = {
 		_x( 'image', 'block search term', 'jetpack' ),
 		_x( 'gallery', 'block search term', 'jetpack' ),
 		_x( 'slider', 'block search term', 'jetpack' ),
+		_x( 'carousel', 'block search term', 'jetpack' ),
```

Comment:
I wonder if this may end up being confusing since we have another Jetpack product named carousel, but doing something different?